tutorial/scripts/LFRic/distributed_memory/4_reductions/helmholtz_solver_alg_mod.x90

```diff
@@ -0,0 +1,87 @@
+!-------------------------------------------------------------------------------
+! Copyright (c) 2017,  Met Office, on behalf of HMSO and Queen's Printer
+! For further details please refer to the file LICENCE.original which you
+! should have received as part of this distribution.
+!-------------------------------------------------------------------------------
+! LICENCE.original is available from the Met Office Science Repository Service:
+! https://code.metoffice.gov.uk/trac/lfric/browser/LFRic/trunk/LICENCE.original
+!-------------------------------------------------------------------------------
+! -----------------------------------------------------------------------------
+! BSD 3-Clause License
+!
+! Modifications Copyright (c) 2017, Science and Technology Facilities Council
```

Comment:
I presume you haven't modified this file for this tutorial and therefore this date is still correct?

Reply:
Yes, it's one of our codes used to test our examples.